boost/meta.yaml

```diff
@@ -14,6 +14,7 @@ source:
 build:
   number: 5
   detect_binary_files_with_prefix: true
+  string: py{{CONDA_PY}}
```

Comment:
Please include the build number in the build string:

```yaml
build:
  {% set build_num = 5 %}
  number: {{build_num}}
  string: py{{CONDA_PY}}_{{build_num}}
```

When conda-build 1.18.3 is finally released, the following syntax will work:

```yaml
build:
  number: 5
  string: py{{CONDA_PY}}_{{PKG_BUILDNUM}}
```